cluster-service/deploy/helm/templates/azure-operators-managed-identities-config.configmap.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
 data:
   azure-operators-managed-identities-config.yaml: |
     controlPlaneOperatorsIdentities:
```

Comment:
We are still missing another control plane operator identity:

The CAPZ one: cluster-api-azure